itou/www/siae_evaluations_views/views.py

```diff
@@ -72,3 +74,51 @@ def siae_job_applications_list(request, template_name="siae_evaluations/siae_job
         "back_url": back_url,
     }
     return render(request, template_name, context)
+
+
+@login_required
+def siae_select_criterias(
+    request, evaluated_job_application_pk, template_name="siae_evaluations/siae_select_criterias.html"
+):
+    siae = get_current_siae_or_404(request)
+    queryset = EvaluatedJobApplication.objects.filter(evaluated_siae__siae=siae)
+
+    evaluated_job_application = get_object_or_404(queryset, pk=evaluated_job_application_pk)
+
+    form_administrative_criteria = AdministrativeCriteriaCommonForm(
+        request.user,
+        siae=siae,
+        administrative_criteria_qs=AdministrativeCriteria.objects.for_evaluated_job_application(
+            evaluated_job_application
+        ),
+        data=request.POST or None,
+    )
+
+    if request.method == "POST" and form_administrative_criteria.is_valid():
+        messages.success(request, "Votre sélection est prise en compte")
+        next_url = reverse("dashboard:index")
+        return HttpResponseRedirect(next_url)
+
+    level_1 = [
+        key
+        for key in form_administrative_criteria.fields.keys()
+        if AdministrativeCriteriaCommonForm.LEVEL_1_PREFIX in key
+    ]
```

Comment:
comme proposé en haut je dirais `[field for field in form.administrative_criteria.fields if field.key.startswith(PREFIX)]` , enfin, si ça fonctionne hein.